goreq_test.go

```diff
@@ -1065,6 +1065,89 @@ func TestRequest(t *testing.T) {
 
 		})
 
+		g.Describe("TLS Proxy", func() {
+			var proxy *httptest.Server
+			proxyCh := make(chan *http.Request, 1)
+
+			createServer := func() {
+				proxy = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+					proxyCh <- r
+					// Implement an entire CONNECT proxy
+					if r.Method == "CONNECT" {
+						// hijacker, ok := w.(http.Hijacker)
```

Comment:
No need to leave it commented you can go ahead and remove it.

Reply:
Oh...Sorry

Reply:
Thank you, I deleted it.